stdlib/locale.pyi

```diff
@@ -34,97 +32,102 @@ if sys.version_info >= (3, 11):
 if sys.version_info < (3, 12):
     __all__ += ["format"]
 
+from _locale import (
+    CHAR_MAX as CHAR_MAX,
+    LC_ALL as LC_ALL,
+    LC_COLLATE as LC_COLLATE,
+    LC_CTYPE as LC_CTYPE,
+    LC_MONETARY as LC_MONETARY,
+    LC_NUMERIC as LC_NUMERIC,
+    LC_TIME as LC_TIME,
+    localeconv as localeconv,
+    setlocale as setlocale,
+    strcoll as strcoll,
+    strxfrm as strxfrm,
+)
+
+if sys.version_info >= (3, 11):
+    from _locale import getencoding as getencoding
+
+# However, some parts of `_local` module is platform-specific:
+if sys.platform == "linux" or sys.platform == "darwin":
```

Comment:
Same comment as above: are you sure these only exist on linux/darwin, and not on any other unix systems?